src/components/balances/AssetsHeader/index.tsx

```diff
@@ -1,20 +1,42 @@
 import { Box } from '@mui/material'
-import type { ReactElement } from 'react'
+import { type ReactElement } from 'react'
 
 import NavTabs from '@/components/common/NavTabs'
 import PageHeader from '@/components/common/PageHeader'
 import CurrencySelect from '@/components/balances/CurrencySelect'
 import { balancesNavItems } from '@/components/sidebar/SidebarNavigation/config'
 
-const AssetsHeader = ({ currencySelect = false }: { currencySelect?: boolean }): ReactElement => {
+import HiddenTokenButton from '../HiddenTokenButton'
+
+const AssetsHeader = ({
```

Comment:
I think we can simplify this component by only passing `children?: ReactNode` instead of the other props since they are not being used in `AssetsHeader`. This would reduce the amount of nesting we have and we could include the child components directly in the balances page like this:

```
<AssetsHeader>
    <Box display="flex" flexDirection="row" alignItems="center" gap={1}>
      {hiddenAssets && (
        <HiddenTokenButton showHiddenAssets={showHiddenAssets} toggleShowHiddenAssets={toggleShowHiddenAssets} />
      )}
      <CurrencySelect />
    </Box>
</AssetsHeader>
```

Reply:
As discussed I will push this small refactor